websites.yml

```diff
@@ -46,3 +46,15 @@ username_patterns:
     invalid_patterns:
       - "\\.(com|net)"
       - "(\\.)\\1{1,}"  # consecutive '.' not allowed
+avatar:
+  pinterest:
```

Comment:
alpha sort the websites?
(it isnt done elsewhere, but that is a cleanup for later.)